src/drivers/uart_esc/uart_esc.cpp

```diff
@@ -0,0 +1,563 @@
+/****************************************************************************
+*
+*   Copyright (c) 2015 Mark Charlebois. All rights reserved.
```

Comment:
@mcharleb This is against our coding standards. Is there a particular reason you chose this approach or would you be ok to adjust?